### PR TITLE
BETA: Register ki.is-a.dev

### PR DESCRIPTION
**Pull request by haruki-py**
Added `ki.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Anandkrishna34** [comment]
for a subdomain

File: domains/ki.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Anandkrishna34",
+        "email": "aanandkrishna344@gmail.com"
+    },
+    "record": {
+        "A": ["69.30.249.53"]
```
Comment (Stef-00012) [inline domains/ki.json:7]: i cannot reach your website
"502 Bad Gateway"

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/ki.json (hunk 1 above)
Comment (Anandkrishna34) [reply domains/ki.json:7]: it is a dbh proxy

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**